### DISCUSSION TITLE
Middlehurst

### DISCUSSION THREAD

**Discussion by subject-matter** [EvgeniyMukhamedjanov/liquid-ajax-cart#69] [Show and tell]
I used LAC to build this site on Shopify

www.middlehurst.co.nz

I also added it to lytteltonlights.co.nz

**Comment by EvgeniyMukhamedjanov** [comment]
Hey! 
Looks awesome!
What should be the link "Made by"?

**Comment by subject-matter** [comment]
Thank you!

Made by Subject Matter

On Mon, 24 Jul 2023 at 10:45 PM, EvgeniyMukhamedjanov <
***@***.***> wrote:

> Hey!
> Looks awesome!
> What should be the link "Made by"?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/EvgeniyMukhamedjanov/liquid-ajax-cart/discussions/69#discussioncomment-6528019>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH357HL22FWALQOUI6XAVG3XRZG6JANCNFSM6AAAAAA2NQK6AI>
> .
> You are receiving this because you authored the thread.Message ID:
> <EvgeniyMukhamedjanov/liquid-ajax-cart/repo-discussions/69/comments/6528019
> @github.com>
>
-- 
Rishi Patel

+64278751837

**Reply by EvgeniyMukhamedjanov** [reply]
Thanks, I've added the websites!
If you wanna fix the credits link, let me know.
